plugins/kotlin-dataframe/testData/box/renameToCamelCase.kt

```diff
@@ -0,0 +1,49 @@
+import org.jetbrains.kotlinx.dataframe.*
+import org.jetbrains.kotlinx.dataframe.annotations.*
+import org.jetbrains.kotlinx.dataframe.api.*
+import org.jetbrains.kotlinx.dataframe.io.*
+
+fun box(): String {
+    // Test DataFrame.renameToCamelCase()
+    val df = dataFrameOf("first_name", "last_name", "user_age")(
+        "John", "Doe", 30
+    )
+    val renamed = df.renameToCamelCase()
+
+    // Verify extension properties
```

Comment:
this is redundant here, no point testing runtime behaviour

Reply:
not entirely, it's still worth checking that the generated property actually has the value we expect it to have. Could catch a potential bugs like types not being recognized or accessors mismatching the correct columns